ch_pipeline/analysis/decorrelation.py

```diff
@@ -21,14 +21,12 @@
   of the timestream data
 """
 # === Start Python 2/3 compatibility
-from __future__ import absolute_import, division, print_function, unicode_literals
 from future.builtins import *  # noqa  pylint: disable=W0401, W0614
 from future.builtins.disabled import *  # noqa  pylint: disable=W0401, W0614
 
 # === End Python 2/3 compatibility
```

Comment:
Presumably this whole thing should go, since `python_requires` is `>=3.9`.

Reply:
Good catch